bake/compose.go

```diff
@@ -124,6 +114,40 @@ func ParseCompose(dt []byte) (*Config, error) {
 	return &c, nil
 }
 
+func loadDotEnv(curenv map[string]string, workingDir string) map[string]string {
+	if curenv == nil {
+		curenv = make(map[string]string)
+	}
+
+	ef, err := filepath.Abs(filepath.Join(workingDir, ".env"))
+	if err != nil {
+		return curenv
+	}
+
+	if _, err := os.Stat(ef); err != nil {
+		return curenv
+	}
+
+	dt, err := os.ReadFile(ef)
+	if err != nil {
+		return curenv
+	}
+
+	envs, err := dotenv.UnmarshalBytes(dt)
+	if err != nil {
+		return curenv
```

Comment:
Is the behavior to ignore parse error the behavior in compose as well?

Reply:
I read the wrong implementation. The right one from compose-go indeed check for errors.